alpenhorn/cli/group/__init__.py

```diff
@@ -17,6 +18,7 @@ def cli():
     """Manage Storage Groups."""
 
 
+cli.add_command(autosync, "auto-sync")
```

Comment:
In response to your question - I personally prefer no dash or underscore (so `autosync` here) because I end up forgetting whether it's a dash or an underscore

Reply:
That's reasonable.  I can change it.